projects/observability/src/pages/apis/service-detail/metrics/service-metrics.dashboard.ts

```diff
@@ -201,6 +201,13 @@ export const serviceMetricsDashboard: DashboardDefaultConfiguration = {
             'color-palette': RED_COLOR_PALETTE,
             'selectable-interval': true,
             'legend-position': LegendPosition.TopLeft,
+            'show-y-axis': true,
+            'y-axis': {
+              type: 'cartesian-axis',
+              'show-grid-lines': true,
+              'min-upper-limit': 25
+            },
+            'max-series-data-points': 150,
```

Comment:
This doesn't appear related, should remain unset/auto.

Reply:
Thanks Aaron, I will remove it from both the places.

Reply:
@aaron-steinfeld Have fixed this, please re-review whenever feasible.